tests/lib/bin/session-tool

```diff
@@ -149,8 +150,34 @@ case "$action" in
 		done
 		exit 0
 		;;
+	has-systemd-and-dbus)
+		# Older systems don't have support for modern sessions
+		# There are two known issues:
+		#  - CentOS 7 and derivatives disabled systemd --user
+		#    https://bugs.centos.org/view.php?id=8767
+		#  - Ubuntu 14.04 with deputy systemd does not connect to DBus
+		#    and systemd-shim is really responding to "systemd" requests.
+		test -n "$(command -v busctl)"         			|| ( echo "no busctl"; exit 1 )
+		if [ "$user" = "root" ]; then
+			# 		/lib										   /usr/lib									   (if present) dbus-broker.service variant
```

Comment:
spacing seems off on this comment?

Reply:
The file uses tabs. not spaces. I could reformat everything but unless I do so this line will look off in various previews with different tab size.

Reply:
could you do it in a follow-up with just the white space change so that I can turn off white space in the diff and approve an empty PR :-)